### PR TITLE
Enhance "amxx" client-side command

### PR DESCRIPTION
**Pull request by Ni3znajomy**
The idea was stolen from SM. SM allows us (as a player) to check what plugins and extensions are running on a server. I think it would be nice to have similar feature in amxx. Also added missing core mode "JIT+ASM32".

### CODE REVIEW

**Comment by Arkshine** [comment]
Is it really useful as simple user to know such informations?

**Comment by Ni3znajomy** [comment]
For simple user, hmm... probably not, but not everyone is simple user. For someone who want to know, what plugin(s) is/are running on a server, it's easier to type "amxx plugins/modules" than write to server's operator to ask and wait, we don't know that he will answer (give a plugin's name or tell "search on your own") or even not answer. I've proposed this to make life easier for people who aren't as you said simple users :smiley: .

**Comment by ClaudiuHKS** [comment]
I think it is, Arkshine. These builds are free, and everyone can have/ use them.
The published AMXX scripts are free as well - see http://www.amxmodx.org/compiler.php

File: amxmodx/meta_api.cpp [hunk 1]
```diff
@@ -958,29 +958,97 @@ void C_ClientCommand(edict_t *pEntity)
...
+						{
+							len += ke::SafeSprintf(&buf[len], sizeof(buf)-len, " (%s)", (*a).getVersion());
+						}
+						if(*(*a).getAuthor())
+						{
+							len += ke::SafeSprintf(&buf[len], sizeof(buf)-len, " by %s", (*a).getAuthor());
+						}
+					
+						len += ke::SafeSprintf(&buf[len], sizeof(buf)-len, " %s\n", (*a).getName());
+                        CLIENT_PRINT(pEntity, print_console, buf);
```
Comment (WildCard65) [inline amxmodx/meta_api.cpp]: I think formatting is not correct for this line.

File: amxmodx/meta_api.cpp [hunk 2]
```diff
@@ -958,29 +958,97 @@ void C_ClientCommand(edict_t *pEntity)
...
+					}
+                    ++a;
```
Comment (WildCard65) [inline amxmodx/meta_api.cpp]: Ditto.

**Comment by WildCard65** [comment]
Looks like you used spaces instead of tabs in your code edits.

**Comment by Ni3znajomy** [comment]
@WildCard65 You're right, I missed that, now it should be ok.

**Comment by IgnacioFDM** [comment]
I like the idea but only if there's a cvar to disable it.

**Comment by WildCard65** [comment]
@IgnacioFDM This PR is based around the very same feature SM has which SM doesn't have a cvar to disable it.

**Comment by IgnacioFDM** [comment]
So what if SM does it that way? If there is no trivial way to disable it, I find this feature invasive.

**Comment by Arkshine** [comment]
I don't mind the idea but I would agree admin should be able to control it.
But then you have already commands for that:  `amx_plugins` and  `amx_modules, in `admincmd.sma`.
Just a matter to change the commands access level.

**Comment by Arkshine** [comment]
Closing this then. If you need to discuss further about this, feel free to do it.

**Pull request closed without merging**